hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/TestS3AProxy.java

```diff
@@ -79,11 +78,11 @@ public void testProxyDefault() throws IOException {
   private void verifyProxy(Configuration proxyConfig,
       boolean isExpectedSecured)
       throws IOException {
-    ClientConfiguration awsConf = new ClientConfiguration();
-    initProxySupport(proxyConfig, "test-bucket", awsConf);
-    Assertions.assertThat(awsConf.getProxyProtocol())
+    ProxyConfiguration config =
+        AWSClientConfig.createProxyConfiguration(proxyConfig, "testBucket");
+    Assertions.assertThat(config.scheme())
         .describedAs("Proxy protocol not as expected")
-        .isEqualTo(isExpectedSecured ? Protocol.HTTPS : Protocol.HTTP);
+        .isEqualTo(isExpectedSecured ? "https" : "https");
```

Comment:
is this intentional?

Reply:
yes, the `Protocol` class was coming from SDK v1, have removed it

Reply:
Sorry, I wasn't clear: it's "https" on both branches.